docs/snippets/tracer/middy.ts

```diff
@@ -4,10 +4,10 @@ import middy from '@middy/core'; // (1)
 const tracer = new Tracer({ serviceName: 'serverlessAirline' });
 
 const lambdaHandler = async (_event: any, _context: any): Promise<void> => {
-    /* ... */
+  /* ... */
```

Comment:
Let's do something with tracer, this time add an annotation like this `tracer.addAnnotation('successfulBooking', true);`